README.md

```diff
@@ -9,7 +9,7 @@ The Web Monetization browser extension is an open source implementation of the W
 
 ### Prerequisites
 
-- [NVM](https://github.com/nvm-sh/nvm) - or another Node Version Manager
+- [NVM](https://github.com/nvm-sh/nvm) (Linux, macOS), [NVM Windows](https://github.com/coreybutler/nvm-windows) (Windows) - or another Node Version Manager
```

Comment:
@dianafulga [mentioned](https://interledgerfoundation.slack.com/archives/C06CVJRT5N3/p1720008488651479?thread_ts=1720008408.321259&cid=C06CVJRT5N3) `nvm install lts/iron` does not work on Windows. I checked docs and they suggest "lts" (not "lts/iron") - so could be that?

Reply:
`nvm install lts` does not work for me with https://github.com/nvm-sh/nvm.

Reply:
https://github.com/interledger/web-monetization-extension/pull/398/commits/e0201d5595bec75b98743061355da82190d84097 This should be better.